### PR TITLE
Drag and drop

### PR DESCRIPTION
**Pull request by caalberts**
sets up the game board, create inventory of scrabble tiles, distribute tiles to players and add drag drop from rack to board.

@cbas please help to review. Thanks

### CODE REVIEW

File: game/img/Scrabble.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<!-- saved from url=(0022)http://localhost:8080/ -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
+    <title>Scrabble</title>
+    <link rel="stylesheet" href="./Scrabble_files/scrabble.css">
+    <link rel="stylesheet" href="./Scrabble_files/dragula.css">
+    <script src="./Scrabble_files/scrabble.js" defer="">
+
+    </script>
+  <style id="style-1-cropbar-clipper">/* Copyright 2014 Evernote Corporation. All rights reserved. */
```
Comment (sebdeckers) [inline game/img/Scrabble.html]: What's this?

File: game/img/Scrabble.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<!-- saved from url=(0022)http://localhost:8080/ -->
```
Comment (sebdeckers) [inline game/img/Scrabble.html]: Err... browser  issue?

File: game/img/Scrabble_files/dragula.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.gu-mirror {
```
Comment (sebdeckers) [inline game/img/Scrabble_files/dragula.css]: Why is this CSS file in the `img` folder?

File: game/img/Scrabble_files/scrabble.css [hunk 4]
```diff
@@ -46,28 +50,65 @@ body {
...
+.rack {
+  align-self: center;
+  background: brown;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+.letter-piece {
+  /*position: absolute;
```
Comment (sebdeckers) [inline game/img/Scrabble_files/scrabble.css]: Leave out the commented code.

File: game/lib/drag-drop.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
+dragula([
+	document.getElementById('1'),
```
Comment (sebdeckers) [inline game/lib/drag-drop.js]: Standard insists on 2 spaces instead of hard tabs.

File: game/lib/drag-drop.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+	// add 'is-moving' class to element being dragged
+	el.classList.add('is-moving');
+})
+.on('dragend', function(el) {
+
+	// remove 'is-moving' class from element after dragging has stopped
+	el.classList.remove('is-moving');
+
+	// add the 'is-moved' class for 600ms then remove it
+	window.setTimeout(function() {
```
Comment (sebdeckers) [inline game/lib/drag-drop.js]: Pretty weird delays. Can it be achieved with `transition-delay`?

File: game/lib/drag-drop.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
+dragula([
```
Comment (sebdeckers) [inline game/lib/drag-drop.js]: Wait a minute... this is not your code. Why aren't you importing it from npm and using browserify to bundle it?

Comment (caalberts) [inline game/lib/drag-drop.js]: sorry ignore this PR. Looks like I accidentally saved a codepen into the folder

**Pull request closed without merging**